jupyter-book/reproducibility/reproducibility.ipynb

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
<p>Not required because a sidebar for these subheaders will also be autogenerated :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>